tasks/main.yml

```diff
@@ -4,6 +4,7 @@
     that:
       - ansible_pkg_mgr is defined
       - ansible_pkg_mgr != ""
+      - ansible_pkg_mgr != "unknown"
```

Comment:
Is "unknown" a return value somewhere? As in, where the value of `ansible_pkg_mgr` is being set? I'm generally confused as to how this could be a valid value.

Reply:
It's a hardcoded value used by Ansible before the package manager is known.